ci-operator/config/ViaQ/vector/ViaQ-vector-release-5.8.yaml

```diff
@@ -28,7 +28,7 @@ base_images:
     namespace: logging
     tag: elasticsearch-proxy
   log-file-metric-exporter:
-    name: 5.y
+    name: "5.8"
```

Comment:
Does it make sense to keep this as `5.y` and build log-file-metric-exporter from master in our release?